README.mkd

```diff
@@ -306,8 +312,9 @@ for readability.
 
 ## TODO
 
-* mdiv, mno-div, mfdiv, mno-fdiv, msave-restore, mno-save-restore, 
-mstrict-align, mno-strict-align, -mexplicit-relocs, -mno-explicit-relocs
+* `mdiv`, `mno-div`, `mfdiv`, `mno-fdiv`, `msave-restore`, 
+  `mno-save-restore`, `mstrict-align`, `mno-strict-align`, 
+  `-mexplicit-relocs`, `-mno-explicit-relocs`
```

Comment:
While touching this, it's probably worth being consistent about `-m` vs `m` (i.e. the `-` prefix)

Reply:
@asb 
Resolved.  As all other occurrences of a command line option used `-` as prefix, force-pushed PR now uses `-` prefix here, too.